hooks/post_gen_project.py

```diff
@@ -1,9 +1,211 @@
+import os
+import json
+import shutil
 import subprocess
+from os.path import join
 
-if __name__ == "__main__":
-{% if cookiecutter.create_repo == "yes" %}
-    subprocess.check_call(['git', 'init'])
-    subprocess.check_call(['hub', 'create'])
-{%- else %}
+import git
+
+from python_terraform import Terraform
+
+try:
+    input = raw_input  # py2 compatibility
+except NameError:
     pass
-{% endif %}
+
+TF_MODULE_PATH = (
+    "git::https://github.com/plus3it/terraform-github-repo.git?ref=0.0.0")
+YES_OPTIONS, NO_OPTIONS = frozenset(["y", "yes"]), frozenset(["n", "no", ""])
+PERM_OPTIONS = frozenset(["push", "pull", "admin"])
+
+
+def multiple_choices(question):
+    selection = []
+    additional = "Would you like to add another (y/n)? [n]: "
+
+    check = True
+    while check:
+        choice = input(question).lower()
+        selection.append(choice)
+        check = check_choice(additional)
+    return selection
+
+
+def check_choice(question):
+    while True:
+        choice = input(question).lower()
+        if choice in YES_OPTIONS:
+            return True
+        if choice in NO_OPTIONS:
+            return False
+
+
+def get_collaborators():
+    # Questions that have multiple layers to them
+    collaborators = []
+    collab_q = "Grant an organizational team access to the repo(y/n)? [n]: "
+
+    if check_choice(collab_q):
+        add_team = True
+
+        while add_team:
+            teams_q = "What team would you like to grant access? "
+            team = input(teams_q).lower()
+
+            if team != "":
+                perm = None
+                while perm not in PERM_OPTIONS:
+                    perms_q = (
+                        "What permissions would you like to grant them"
```

Comment:
python will automatically concat strings when wrapped in parens...
```suggestion
                        "What permissions would you like to grant them "
                        "(push, pull, admin)? ")
```